apps/adoption/src/app/components/stackblitz/templates/core-latest.ts

```diff
@@ -0,0 +1,130 @@
+<div class="main-container">
+  <header class="header-2">
+    <div class="branding">
+      <a class="nav-link">
+        <clr-icon shape="home" size="24"></clr-icon>
+        <span class="title">Clarity</span>
+      </a>
+    </div>
+    <div class="header-nav">
+      <a class="active nav-link nav-text">Home</a>
+    </div>
+  </header>
+  <div class="content-container">
+    <div class="content-area">
+      <app-root>loading</app-root>
+    </div>
+  </div>
+</div>
+`,
+  'src/styles.css': '',
+  'src/polyfills.ts': `import 'zone.js/dist/zone';`,
+
+  'src/main.ts': `
+import './polyfills';
+
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+
+platformBrowserDynamic().bootstrapModule(AppModule).then(ref => {
+  // Ensure Angular destroys itself on hot reloads.
+  if (window['ngRef']) {
+    window['ngRef'].destroy();
+  }
+  window['ngRef'] = ref;
+
+  // Otherwise, log the boot error
+}).catch(err => console.error(err));
+`,
+
+  'src/app/app.module.ts': `
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { ClarityModule } from '@clr/angular';
+
+import '@clr/icons';
+
+import { AppComponent } from './app.component';
+
+import '@cds/core/alert/register.js';
+import { CdsModule } from '@cds/angular';
+
+@NgModule({
+  imports:      [ BrowserModule, FormsModule, ClarityModule, BrowserAnimationsModule, CdsModule ],
+  declarations: [ AppComponent ],
+  bootstrap:    [ AppComponent ]
+})
+export class AppModule { }
+`,
+
+  'src/app/app.component.ts': `
+import { Component } from '@angular/core';
+@Component({
+  selector: 'app-root',
+  template: \`
+  <h3>Clarity Starter Instructions:</h3>
+  <ul>
+    <li>Implement the problem in the new editor.</li>
+    <li>Save the result, and share the url in a GitHub or StackOverflow issue.</li>
+  </ul>
+\`,
+})
+export class AppComponent  {
+  name = 'Angular';
+}`,
+};
+
+/**
+ * Make sure that there are no new line on top and bottom of the files
+ */
+Object.keys(files).forEach((filename: string) => {
+  files[filename] = files[filename].trim();
+});
+
+const project: StackblitzProject = {
+  title: 'Clarity Demo Application',
+  description: 'Clarity auto-generated application',
+  template: 'angular-cli',
+  tags: ['clarity'],
+  files: files,
+  dependencies: {
+    '@angular/animations': '10.2.4',
+    '@angular/common': '10.2.4',
+    '@angular/compiler': '10.2.4',
+    '@angular/core': '10.2.4',
+    '@angular/forms': '10.2.4',
+    '@angular/platform-browser': '10.2.4',
+    '@angular/platform-browser-dynamic': '10.2.4',
+    '@angular/router': '10.2.4',
+    '@cds/angular': '5.0.2',
+    '@cds/core': '5.0.2',
+    '@clr/angular': '4.0.11',
+    '@clr/core': '4.0.11',
```

Comment:
Why clr-core AND cds-core...? That could result in some weird collisions...

Reply:
You are right this probably was left from some of my experiments to make Stackblitz work .. I was trying to make them work with older version - will still have hard time making Stackblitz v5 work without NGCC error